src/assets/server/rest/v1/schemas/common/common.json

```diff
@@ -6,11 +6,31 @@
       "pattern": "^$|^[0-9a-fA-F]{24}$",
       "sanitize": "mongo"
     },
-    "IDs": {
+    "ids": {
       "type": "string",
       "pattern": "^$|^([0-9a-fA-F]{24}\\|?)*$",
       "sanitize": "mongo"
     },
+    "string-id": {
+      "type": "string",
+      "pattern": "^[^|]*$",
+      "sanitize": "mongo"
+    },
+    "string-ids": {
+      "type": "string",
+      "pattern": "^(([^|]*)(\\||[^|])*)?$",
+      "sanitize": "mongo"
+    },
+    "number-id": {
+      "type": "number",
+      "pattern": "^(\\d)*$",
+      "sanitize": "mongo"
+    },
+    "number-ids": {
+      "type": "number",
```

Comment:
Type string, as it will be separated by pipes

Reply:
Okay, fixed.